provision.py

```diff
@@ -105,6 +107,11 @@
 
 def main():
     run(["sudo", "apt-get", "update"])
+    run(["sudo", "apt-get", "-y", "install", "software-properties-common"])
+    run(["sudo", "add-apt-repository", "-y", "universe"])
+    run(["sudo", "add-apt-repository", "-y", "ppa:groonga/ppa"])
+
+    run(["sudo", "apt-get", "update"])
```

Comment:
I think that addressing this is required to make pgroonga a supported feature for Zulip, but I'm OK with making this a follow-up issue if you like.  But we should clearly mark this feature as experimental in the docs/ page and be clear it only works in the Zulip development environment for now.  Does that seem reasonable?  I don't want to have people think they can use this in production when that's not done yet.